tools/editor/plugins/resource_preloader_editor_plugin.cpp

```diff
@@ -60,36 +60,40 @@ void ResourcePreloaderEditor::_notification(int p_what) {
 	}
 }
 
-void ResourcePreloaderEditor::_file_load_request(const String& p_path) {
+void ResourcePreloaderEditor::_files_load_request(const Vector<String>& p_path) {
 
+	for(int i=0;i<p_path.size();i++) {
 
-	RES resource;
+		String path = p_path[i];
 
-	resource = ResourceLoader::load(p_path);
+		RES resource;
+		resource = ResourceLoader::load(path);
 
-	if (resource.is_null()) {
-		dialog->set_text("ERROR: Couldn't load resource!");
-		dialog->set_title("Error!");
-		//dialog->get_cancel()->set_text("Close");
-		dialog->get_ok()->set_text("Close");
-		dialog->popup_centered_minsize();
-		return; ///beh should show an error i guess
-	}
+		if (resource.is_null()) {
```

Comment:
for(int i=0;i<p_paths.size();i++) {
		String path = p_paths[i];
		RES resource;
		resource = ResourceLoader::load(path);
		if (resource.is_null()) { // two tabs indent is right.